consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DataUseLetterResource.java

```diff
@@ -125,12 +125,15 @@ public Consent updateDUL(
     @GET
     @Produces(MediaType.APPLICATION_OCTET_STREAM)
     @RolesAllowed({"ADMIN","CHAIRPERSON","MEMBER","DATAOWNER"})
-    public Response getDUL(@PathParam("id") String consentId) {
+    public Response getDUL(@PathParam("id") String consentId, @QueryParam("electionId") Integer electionId) {
         String msg = String.format("GETing Data Use Letter for consent with id '%s'", consentId);
```

Comment:
you might want to add in the election id to the debug statement

Reply:
Adding election id to debug statement.